fhir2dataset/api_caller.py

```diff
@@ -203,57 +202,38 @@ def _flatten_item_results(self, lines):
     @timing
     def _get_match_search(self, json_resource) -> dict:
         lines = self._init_data()
-        for element, search in self.expressions.items():
-            item = self._search(search, json_resource)
-            lines[element].extend(item)
+        item = multiple_search_dict(
+            [json_resource["resource"]], list(self.searchparam_fhirpath.values())
+        )
+        for idx, fhirpath in enumerate(self.searchparam_fhirpath.keys()):
+            lines[fhirpath].extend(item[0][idx])
```

Comment:
thanks to the new `dataclasses`, I can update almost directly the object elements in the javascript code. This way, we can do without the `lines` objects.
Nevertheless, I kept the `_init_data()` because the dataframe must still be built even if there is no corresponding result.